griptape/drivers/prompt_model/sagemaker_llama_prompt_model_driver.py

```diff
@@ -8,34 +8,40 @@
 
 @define
 class SageMakerLlamaPromptModelDriver(BasePromptModelDriver):
-    DEFAULT_MAX_TOKENS = 600
+    # Default context length for all Llama 3 models is 8K as per https://huggingface.co/blog/llama3
+    DEFAULT_MAX_TOKENS = 8000
```

Comment:
Can you please rename this to `DEFAULT_MAX_INPUT_TOKENS` for consistency with the Tokenizer classes?